### PR TITLE
Removes 'members' link from header

### PR DESCRIPTION
**Pull request by joaojusto**
Removes 'members' from top navbar and puts it on the sidebar

### CODE REVIEW

File: app/assets/javascripts/app/discussions/index.html.slim [hunk 1]
```diff
@@ -6,7 +6,7 @@ div ui-view="sidebar"
       .md-toolbar-tools layout="row" layout-align="space-between center"
         a.IconNavigation href="" ng-click="ctrl.toggleSidebar()"
         a href="#/discussions" Council
-        a href="#/members" Members
+        a href="#"
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/index.html.slim]: What is this line for?

**Comment by gabrielpoca** [comment]
@joaojusto where does this task come from? I don't see any "Members" on the design

File: app/assets/javascripts/app/discussions/index.html.slim (hunk 1 above)
Comment (joaojusto) [inline app/assets/javascripts/app/discussions/index.html.slim]: It was a 'hack' for the flex box maintain the logo centered but I have found a better way, I´ll change that

**Comment by joaojusto** [comment]
This was discussed by me and João in person and we have created the card 2 or 3 days ago
https://trello.com/c/fpUeelRq/46-remove-members-from-navbar

**Comment by naps62** [comment]
@gabrielpoca there was no design for Members in the first place. I added that page because it was requested, although it's even finished yet

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**